repose-aggregator/components/filters/keystone-v2-filter/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -49,6 +49,13 @@
             <xs:element name="require-service-endpoint" type="ServiceEndpointType" minOccurs="0" maxOccurs="1"/>
             <xs:element name="pre-authorized-roles" type="RolesList" minOccurs="0" maxOccurs="1"/>
         </xs:all>
+        <xs:attribute name="ignored-roles" type="xs:string" default="identity:tenant-access" use="optional">
```

Comment:
Out of curiosity, since this is a space-separated list, why not use the XML Schema `list` type?

Reply:
mostly because that behaves super weirdly with defaults.

Reply:
Oh, yeah, I forgot about that. Freakin' JAXB.